pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -50,7 +50,9 @@ type ClusterHealth struct {
 
 const (
 	// OperatorSettingConfigMapName refers to ConfigMap that configures rook ceph operator
-	OperatorSettingConfigMapName string = "rook-ceph-operator-config"
+	OperatorSettingConfigMapName   string = "rook-ceph-operator-config"
+	EnforceHostNetworkSettingName  string = "ROOK_ENFORCE_HOST_NETWORK"
+	EnforceHostNetworkDefaultValue string = "false"
```

Comment:
do we need this to be an exported variable? its not used outside of this function

Reply:
it's good to have it as not exposed unless it's required.

Reply:
@Madhu-1 
> it's good to have it as not exposed unless it's required.

Good point. unexported. and updated the PR